### PR TITLE
ProgressSimple React

### PR DESCRIPTION
**Pull request by jasperfurniss**
#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUX-721

#### Checklist:
- [ ] **SPECS** Please cover your changes with specs
- [ ] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx [hunk 1]
```diff
@@ -1,21 +1,74 @@
...
-    return (
-      <div className="pb_progress_simple">
-        <span>{'PROGRESS SIMPLE CONTENT'}</span>
-      </div>
-    )
+const ProgressSimple = ({
+  align,
+  className,
+  dark = false,
+  max,
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx]: On any of the props that are required (they don't have `?`) we will want to ensure some default is set

File: app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx [hunk 2]
```diff
@@ -1,21 +1,74 @@
...
+          <div
+              className="progress_simple_value"
+              style={valueStyles}
+          />
+          <Else />
+          <div
+              className="progress_simple_value"
+              style={nonPercentStyles}
+          />
+        </If>
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx]:
```suggestion
        <div
              className="progress_simple_value"
              style={valueStyles}
         />
```

Can we consolidate `valueStyles` and `nonPercentStyles` into one variable that uses `percent` to determine the output here? (see above suggestion)

File: app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx [hunk 3]
```diff
@@ -1,21 +1,74 @@
...
+}: ProgressSimpleProps) => {
+  const styles = {
+    width: width,
   }
-}
 
-ProgressSimple.propTypes = propTypes
+  const valueStyles = {
+    width: `${percent}%`,
+  }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx]:
```suggestion
  const valueStyles = {
    width: percent ? `${percent}%` : `${(value * 100) / max}%`,
  }
```

File: app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx [hunk 4]
```diff
@@ -1,21 +1,74 @@
...
+
+  const nonPercentStyles = {
+    width: `${(value * 100) / max}%`,
+  }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx]:
```suggestion
```

**Comment by thestephenmarshall** [comment]
You will also need to update the `CHANGELOG.md`

File: Gemfile.lock [hunk 5]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    playbook_ui (4.7.0)
+    playbook_ui (4.7.1)
```
Comment (jasoncypret) [inline Gemfile.lock]: ??? This shouldn't be included in yours

File: lib/playbook/version.rb [hunk 6]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Playbook
-  VERSION = "4.7.0"
+  VERSION = "4.7.1"
```
Comment (jasoncypret) [inline lib/playbook/version.rb]: ???

File: package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "playbook-ui",
-  "version": "4.7.0",
+  "version": "4.7.1",
```
Comment (jasoncypret) [inline package.json]: ???

File: spec/pb_kits/playbook/kits/form_pill_spec.rb [hunk 8]
```diff
@@ -8,8 +8,6 @@
   it { is_expected.to define_partial }
```
Comment (jasoncypret) [inline spec/pb_kits/playbook/kits/form_pill_spec.rb]: I don't think up updated this

**Pull request merged**